packages/dashboard/src/app/views/myStories/filters/reducer.js

```diff
@@ -35,34 +35,49 @@ const reducer = (state, { type, payload = {} }) => {
   switch (type) {
     case types.UPDATE_FILTER: {
       const { key, value } = payload;
-      const filter = state.filters.find((f) => f.key === key);
+      const _filter = state.filters.find((filter) => filter.key === key);
```

Comment:
Why change this?

Reply:
This was updated inaccodance with the other comments by @sblinde on `filters/reducer` to be more readable. [https://github.com/GoogleForCreators/web-stories-wp/pull/11736#discussion_r900405598, https://github.com/GoogleForCreators/web-stories-wp/pull/11736#discussion_r900405907, etc]

Reply:
I don't feel like adding an underscore here makes it more readable. I would leave it as it was and just change line 56.

Reply:
@spacedmonkey My feedback was about not using letters alone as arguments within a function and to use a more descriptive word. It's not the change of adding an underscore that was suggested, but it is a way to tackle the change I suggested so that `filter` isn't declared as upper scope and so the argument is more descriptive than `f`.

Reply:
I understand your suggestion as well, but just want to note that it's important to me that we leave our code readable and understood by whoever may pick up on it in the future, so it's feedback I will always give to my team when I'm able to.